sdks/java/io/kafka/src/test/java/org/apache/beam/sdk/io/kafka/KafkaIOIT.java

```diff
@@ -258,6 +301,17 @@ private void cancelIfTimeouted(PipelineResult readResult, PipelineResult.State r
         .withTopic(options.getKafkaTopic());
   }
 
+  private KafkaIO.Read<byte[], String> readFromKafkaNullKey() {
+    return KafkaIO.<byte[], String>read()
+        .withSupportsNullKeys()
+        .withBootstrapServers(options.getKafkaBootstrapServerAddresses())
+        .withConsumerConfigUpdates(ImmutableMap.of("auto.offset.reset", "earliest"))
+        .withTopic(options.getKafkaTopic())
+        .withMaxNumRecords(100)
```

Comment:
Usually hardcoding a number in a util function is not perferred. If your test is the only place using this, it might be better to have the test construct Kafka read directly.